lms/djangoapps/philu_overrides/branding/api.py

```diff
@@ -43,7 +43,7 @@ def get_auth_footer(is_secure=True):
     }
 
 
-def get_non_auth_footer(is_secure=True):
+def get_non_auth_footer(is_secure=True):  # pylint: disable=unused-argument
```

Comment:
Here as well, can we remove the unused parameter, by first examining their usage, and removing the parameter from the function calls as well?